src/components/gltf-model.js

```diff
@@ -0,0 +1,61 @@
+var registerComponent = require('../core/component').registerComponent;
+var THREE = require('../lib/three');
+
+var GLTFLoader = THREE.glTFLoader; // To appease the linter
+var loader = new GLTFLoader();
+var camera = null;
+var gltf = null;
+
+module.exports.Component = registerComponent('gltf-model', {
+  schema: {
+    type: 'src'
+  },
+
+  init: function () {
+    this.model = null;
+  },
+
+  update: function () {
+    var self = this;
+    var el = this.el;
+    var src = this.data;
+
+    camera = this.el.sceneEl.camera;
+    if (!src) { return; }
+
+    this.remove();
+
+    loader.load(src, function (gltfModel) {
+      var i = null;
+      var len = null;
+      var animation = null;
+
+      gltf = gltfModel;
+
+      self.model = gltfModel.scene;
+      el.setObject3D('mesh', self.model);
+      el.emit('model-loaded', {format: 'gltf', model: self.model});
+
+      if (gltf.animations && gltf.animations.length) {
+        len = gltf.animations.length;
+        for (i = 0; i < len; i++) {
+          animation = gltf.animations[i];
+          animation.loop = true;
+          animation.play();
+        }
+      }
+    });
+  },
+
+  remove: function () {
+    if (!this.model) { return; }
+    this.el.removeObject3D('mesh');
+  },
+
+  tick: function () {
+    if (gltf && camera) {
+      THREE.glTFAnimator.update();
```

Comment:
Ah sorry, meant to say it needs to be called exactly once no matter how many models there are (not that it's already called somewhere else). Does it work if you put two glTF models into the same scene, or do the animations start going twice as fast? I don't think THREE.AnimationMixer is called anywhere either, yet, but I don't know.